src/components/Navbar/GNB.jsx

```diff
@@ -0,0 +1,37 @@
+import SmallMainLogo from "assets/icon/main_logo_small.svg";
+import MainLogo from "assets/icon/main_logo.svg";
+import { useLocation } from "react-router-dom";
+import { Button } from "components/Button";
+import useDeviceState from "hooks/useDeviceState";
+import { NavLink } from "react-router-dom";
+import * as S from "./Navbar.style";
+
+export default function GNB() {
+  const { pathname } = useLocation();
+  const { isMobileWidth } = useDeviceState();
+
+  return (
+    <S.NavbarContainer>
+      <NavLink to="/">
+        <img src={isMobileWidth ? SmallMainLogo : MainLogo} alt="main-logo" />
+      </NavLink>
+      <S.NavbarLinkContainer>
+        <NavLink
+          to="/free-board"
+          style={({ isActive }) => ({ color: isActive ? "var(--blue)" : "" })}
+        >
+          자유게시판
+        </NavLink>
+        <NavLink
+          to="/items"
+          style={({ isActive }) => ({
+            color: isActive || pathname === "/additem" ? "var(--blue)" : "",
+          })}
+        >
+          중고마켓
+        </NavLink>

```

Comment:
흠 이런것도 styled component로 만들어서 isActive prop만 넘겨주는 형태는 어때요?
jsx는 가독성을 위해 최대한 깔끔하게 보여지도록 신경써주면 좋답니다 :)

```suggestion
        <StyledNavLink
          to="/free-board"
          isActive={isActive}
        >
          자유게시판
        </StyledNavLink>
        <StyledNavLink
          to="/items"
          isActive={isActive || pathname === "/additem"}
        >
          중고마켓
        </StyledNavLink>

```